apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/config/MetricsConfig.java

```diff
@@ -60,13 +60,22 @@ interface TenantRequestCounterConfig {
     @WithDefault("error")
     String errorTag();
 
-    /** @return The tag key for user-agent flag, defaults to <code>error</code>. */
+    /** @return The tag key for user-agent flag, defaults to <code>user_agent</code>. */
     @NotBlank
     @WithDefault("user_agent")
     String userAgentTag();
 
     /** @return If tenant counting metric should include the user agent information. */
     @WithDefault("false")
     boolean userAgentTagEnabled();
+
+    /** @return The tag key for HTTP status flag, defaults to <code>status</code>. */
+    @NotBlank
+    @WithDefault("status")
+    String statusTag();
+
+    /** @return If tenant counting metric should include the HTTP status information. */
+    @WithDefault("true")
```

Comment:
Just to make sure: we do want to enable this by default? I guess there's little risk for explosion of values since number of http response codes is relatively samll.